File: async-scala-client/src/main/scala/io/swagger/client/model/Transport_volumes.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class Transport_volumes (
  qVol: Option[String],  // Quantity of packages transported
specie: Option[String],  // package type
brand: Option[String],  // brand
volumeNumeration: Option[String],  // packages numeration
netWeight: Option[Double],  // net weight using Kg
grossWeight: Option[Double],  // gross weight using Kg
seal: Option[List[String]]
)

File: javascript-closure-angular-client/API/Client/TransportVolumes.js
goog.provide('API.Client.Transport_volumes');

/**
 * @record
 */
API.Client.TransportVolumes = function() {}

/**
 * Quantity of packages transported
 * @type {!string}
 * @export
 */
API.Client.TransportVolumes.prototype.qVol;

/**
 * package type
 * @type {!string}
 * @export
 */
API.Client.TransportVolumes.prototype.specie;

/**
 * brand
 * @type {!string}
 * @export
 */
API.Client.TransportVolumes.prototype.brand;

/**
 * packages numeration
 * @type {!string}
 * @export
 */
API.Client.TransportVolumes.prototype.volumeNumeration;

/**
 * net weight using Kg
 * @type {!number}
 * @export
 */
API.Client.TransportVolumes.prototype.netWeight;

/**
 * gross weight using Kg
 * @type {!number}
 * @export
 */
API.Client.TransportVolumes.prototype.grossWeight;

/**
 * @type {!Array<!string>}
 * @export
 */
API.Client.TransportVolumes.prototype.seal;

